enhancements/operator-hub-check.md

```diff
@@ -188,10 +196,29 @@ The Kubernetes `1.22` release will impact the Pipeline. After the implementation
 We could return an error optionally:
 - `minKubeVersion >= 1.22 or empty/nil`  return the error result.
 
-All bundles which are using the no longer supported Kubernetes API will start to fail in the above check which indeed is the purpose of this proposal. However, the risk still solved since the Pipeline can indeed inform a Kubernetes version lower such as `--optional-values="k8s=1.21"` if required until all projects are properly update.
+All bundles which are using the no longer supported Kubernetes API will start to fail in the above check which indeed is the purpose of this proposal. However, the risk still solved since the Pipeline can indeed inform a Kubernetes version lower such as `--optional-values="k8s-version=1.21"` if required until all projects are properly update.
 
 Then, this proposal has a very low risks and/or mitigations which are usually faced to address any change. However, see that the usage of the validator can be easily disabled from any place in case it is required until a bug fix can be patched as well. For example, the user or a job can decide not use the command until a bug fix to be addressed.
 
+#### (Optional) Suggested Follow-ups
+
+Following the proposed follow-ups for this implementation which are not required to address its requirements and uses cases and are more related to the additional option added here which allows to pass values via a []map of key/values to the validators. 
+
+
+The `k8s-version` key, besides at this moment be only be used by the validator `OperatorHub.io`, requires to be "global". Otherwise, if it starts to be used for another validator would not be possible to use it as: 
+
+ ```sh
+ $ operator-sdk bundle validate ./bundle \
+    --select-optional suite=operatorframework \
+    --optional-values="k8s-version=1.22"
+ ```
+
+However, we could also have specific keys for each validator and then, to address this need would be nice define something like terraform’s [TF_VAR_name][tf-var] for validators, where the prefix is the validator name capitalized, so you don’t have to pass in some object or define flags for the validator lib consumer.
```

Comment:
I think after some discussion it was decided to have an optional colon-delimited prefix per-validator or per-suite, ex. the validator `operatorhub:k8s-version` vs. the suite `operatorframework:k8s-version` vs. global `k8s-version`.

/cc @fabianvf @jmrodri

Reply:
@estroz have we the suite tests implemented in the API or is that this is done only on the sdk side?

Reply:
HI @estroz, 

Since it these improvements are a little out of the scope of the proposal and our idea here, for now, is keep tracked all discussion made so far and not define that; I added your comment in the ep as well. Please, let me know what do you think and if we can move forward with it.